saritasa_invocations/django.py

```diff
@@ -388,7 +389,24 @@ def load_django_remote_env_db_settings(
 
     env_config = decouple.Config(decouple.RepositoryEnv(env_path))
     pathlib.Path(env_path).unlink()
+    if database_url := str(
```

Comment:
Please describe in docstring that it support 2 sources:
* single env variable from `config.django.remote_db_url_config_name`
* multiple env vars from `config.django.remote_db_config_mapping`